code/modules/buildmode/submodes/fill.dm

```diff
@@ -51,10 +53,19 @@
 			// if there's an analogous proc for this on tg lmk
 			// empty_region(block(get_turf(cornerA),get_turf(cornerB)))
 		else
+			var/selection_size = abs(cornerA.x - cornerB.x) * abs(cornerA.y - cornerB.y)
+
+			if(selection_size > FILL_WARNING_MIN) // Confirm fill if the number of tiles in the selection is greater than FILL_WARNING_MIN
+				var/choice = alert("Your selected area is [num2text(selection_size)] tiles! Continue?", "Large Fill Confirmation", "Yes", "No")
```

Comment:
```suggestion
				var/choice = alert("Your selected area is [selection_size] tiles! Continue?", "Large Fill Confirmation", "Yes", "No")
```
I didn't pay attention properly, but on this syntax numbers will be converted by default, the proc is just boilerplate.